pkg/network/setup/netstat.go

```diff
@@ -66,23 +66,19 @@ func (c *NetStat) UpdateStatus(vmi *v1.VirtualMachineInstance, domain *api.Domai
 		return nil
 	}
 
-	if len(vmi.Status.Interfaces) == 0 {
-		// Set Pod Interface
-		interfaces := make([]v1.VirtualMachineInstanceNetworkInterface, 0)
-		for _, network := range vmi.Spec.Networks {
-			podIface, err := c.getPodInterfacefromFileCache(vmi, network.Name)
-			if err != nil {
-				return err
-			}
+	vmi.Status.Interfaces = []v1.VirtualMachineInstanceNetworkInterface{}
+	for _, network := range vmi.Spec.Networks {
+		podIface, err := c.getPodInterfacefromFileCache(vmi, network.Name)
+		if err != nil {
+			return err
+		}
 
-			ifc := v1.VirtualMachineInstanceNetworkInterface{
-				Name: network.Name,
-				IP:   podIface.PodIP,
-				IPs:  podIface.PodIPs,
-			}
-			interfaces = append(interfaces, ifc)
+		ifc := v1.VirtualMachineInstanceNetworkInterface{
+			Name: network.Name,
+			IP:   podIface.PodIP,
+			IPs:  podIface.PodIPs,
 		}
-		vmi.Status.Interfaces = interfaces
+		vmi.Status.Interfaces = append(vmi.Status.Interfaces, ifc)
```

Comment:
I cannot add the comment to the exact place due to github limitiation, but this comment -

> // This calculates the vmi.Status.Interfaces based on the following data sets:
> // - vmi.Status.Interfaces - previously calculated interfaces, this can contain data (pod IP)
> //   set in the previous loops (when there are no interfaces), which can not be deleted,

should be updated.

Reply:
Done